npc/cities/aldebaran.txt

```diff
@@ -1173,220 +1171,220 @@ aldeba_in,88,161,3	script	Kafra#05	4_F_KAFRA3,{
-		if(@Lotto == 4) {
-			getitem Yggdrasilberry,2;//Items: Yggdrasilberry,
-			getitem Seed_Of_Yggdrasil,1;//Items: Yggdrasil_Seed,
+		mes("[Kafra]");
+		mes("WOW!!!!..... You win!!! 1st Prize~! Congratulations~~ You got the 1st prize~~");
+		if (@Lotto == 1) getitem(Wooden_Mail, 1);//Items: Wooden_Mail,
+		if (@Lotto == 2) getitem(Mantle, 1);//Items: Mantle,
+		if (@Lotto == 3) getitem(Berserk_Potion, 10);//Items: Berserk_Potion,
+		if (@Lotto == 4) {
+			getitem(Yggdrasilberry, 2);//Items: Yggdrasilberry,
+			getitem(Seed_Of_Yggdrasil, 1);//Items: Yggdrasil_Seed,
 		}
-		if(@Lotto == 5) getitem Yggdrasilberry,3;//Items: Yggdrasilberry,
+		if (@Lotto == 5) getitem(Yggdrasilberry, 3);//Items: Yggdrasilberry,
 		return;
 	sL_Prize2:
-		mes "[Kafra]";
-		mes "Oh! WOW! You've won the 2nd prize! Congratulations!!";
-		if(@Lotto == 1) getitem Shoes,1;//Items: Shoes,
-		if(@Lotto == 2) getitem Cap,1;//Items: Cap,
-		if(@Lotto == 3) getitem Sunglasses,1;//Items: Sunglasses,
-		if(@Lotto == 4) getitem Royal_Jelly,3;//Items: Royal_Jelly,
-		if(@Lotto == 5) {
-			getitem Seed_Of_Yggdrasil,1;//Items: Yggdrasil_Seed,
-			getitem Royal_Jelly,10;//Items: Royal_Jelly,
+		mes("[Kafra]");
+		mes("Oh! WOW! You've won the 2nd prize! Congratulations!!");
+		if (@Lotto == 1) getitem(Shoes, 1);//Items: Shoes,
+		if (@Lotto == 2) getitem(Cap, 1);//Items: Cap,
+		if (@Lotto == 3) getitem(Sunglasses, 1);//Items: Sunglasses,
+		if (@Lotto == 4) getitem(Royal_Jelly, 3);//Items: Royal_Jelly,
+		if (@Lotto == 5) {
+			getitem(Seed_Of_Yggdrasil, 1);//Items: Yggdrasil_Seed,
+			getitem(Royal_Jelly, 10);//Items: Royal_Jelly,
 		}
 		return;
 	sL_Prize3:
-		mes "[Kafra]";
-		mes "Congratulations! You've won the 3rd prize.";
-		if(@Lotto == 1) getitem Wing_Of_Butterfly,4;//Items: Butterfly_Wing,
-		if(@Lotto == 2) getitem Blue_Potion,3;//Items: Blue_Potion,
-		if(@Lotto == 3) getitem Glasses,1;//Items: Glasses,
-		if(@Lotto == 4) getitem White_Potion,15;//Items: White_Potion,
-		if(@Lotto == 5) getitem White_Potion,30;//Items: White_Potion,
+		mes("[Kafra]");
+		mes("Congratulations! You've won the 3rd prize.");
+		if (@Lotto == 1) getitem(Wing_Of_Butterfly, 4);//Items: Butterfly_Wing,
+		if (@Lotto == 2) getitem(Blue_Potion, 3);//Items: Blue_Potion,
+		if (@Lotto == 3) getitem(Glasses, 1);//Items: Glasses,
+		if (@Lotto == 4) getitem(White_Potion, 15);//Items: White_Potion,
+		if (@Lotto == 5) getitem(White_Potion, 30);//Items: White_Potion,
 		return;
 	sL_Prize4:
-		mes "[Kafra]";
-		mes "You've won the 4th prize.";
-		if(@Lotto == 1) getitem Sweet_Potato,100;//Items: Sweet_Potato,
-		if(@Lotto == 2) getitem Red_Potion,150;//Items: Red_Potion,
-		if(@Lotto == 3) getitem Orange_Potion,150;//Items: Orange_Potion,
-		if(@Lotto == 4) getitem Blue_Potion,5;//Items: Blue_Potion,
-		if(@Lotto == 5) getitem Blue_Potion,10;//Items: Blue_Potion,
+		mes("[Kafra]");
+		mes("You've won the 4th prize.");
+		if (@Lotto == 1) getitem(Sweet_Potato, 100);//Items: Sweet_Potato,
```

Comment:
1) getitem in new line.
2) Comment of item name no longer needed, as constants are used